static/js/narrow.js

```diff
@@ -268,23 +268,72 @@ export function activate(raw_operators, opts) {
         const adjusted_operators = [];
         let operators_changed = false;
 
+        function message_history_match_narrow(narrow_topic, narrow_stream_id, message) {
+            let current_topic = message.topic;
+            let current_stream_id = message.stream_id;
+
+            // check current message values
+            if (narrow_topic === current_topic && narrow_stream_id === current_stream_id) {
```

Comment:
Topic comparisons in Zulip are technically case-insensitive; we should probably respect that here. `util.same_stream_and_topic` is an existing function for doing such comparisons; using that might suggest it's a good idea to create a little object rather than having two variables?  Not sure.

Reply:
Thanks for pointing out that existing function. I kept the two separate parameters in the function (`stream_id` and `topic`), but it could be easily updated to take an object instead. I felt like it was clearer and potentially more useful with the two separate parameters, but I could be mistaken.